src/modules/common/components/bars/topBar/network.css

```diff
@@ -68,3 +68,7 @@
 .offline {
   background-color: red;
 }
+
+.tooltipContainer {
+  transform: translateX(-24px) !important;
```

Comment:
We shouldnt use important for new code 🙏 . Increase the specificity by making it more specific using parent classes. 
For example, putting it inside message will make it more specific. 

```
& > .message {
  & .tooltipContainer {
    transform: translateX(-24px) 
```

Will make it much easier to maintain the CSS